Misc/NEWS

```diff
@@ -10,6 +10,10 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-30876: Relative import from unloaded package now reimports the package
```

Comment:
If you already have a version of blurb installed, you may need `--upgrade` to get it to upgrade to the latest one.